CONTRIBUTING.md

```diff
@@ -90,6 +90,35 @@ pants run libs/next_gen_ui_llama_stack/agent_test.py
 pants fmt lint check ::
 ```
 
+### Dependency Management
+
+Dependencies are controled by Pants build. All 3rd party dependencies are managed in [libs/3rdparty/python/](./libs/3rdparty/python/) 
+directory and categorized by frameworks.
+During build/tests Pants automatically detects your dependencies from your source code and creates your dependency tree.
+
+To check your dependencies you can run `pants dependencies --transitive tests/ai_eval_components/types.py` to see your dependencies.
```

Comment:
This shows dependencies necessary for that one file (types.py), or whole module?